### ISSUE TITLE
wikipedia_en_all_maxi has failed

### ISSUE THREAD

**Issue by benoit74** [openzim/zim-requests#925]
Recipe: https://farm.openzim.org/recipes/wikipedia_en_all_maxi

```
[error] [2024-03-16T12:45:16.673Z] Error downloading article Katie_Rogers_(journalist)
[error] [2024-03-16T13:04:08.405Z] Error downloading article Partha_Pratim_Bora
[error] [2024-03-16T13:53:58.475Z] Unable to retrieve js/css dependencies for article 'Sadist (band)': internal_api_error_Wikimedia\RequestTimeout\RequestTimeoutException
[error] [2024-03-16T13:53:58.475Z] Error downloading article Sadist_(band)
[error] [2024-03-16T13:53:58.475Z] Failed to run mwoffliner after [157963s]: {
	"name": "Error",
	"message": "Unable to retrieve js/css dependencies for article 'Sadist (band)': internal_api_error_Wikimedia\\RequestTimeout\\RequestTimeoutException"
}
[error] [2024-03-16T13:53:58.475Z] 

**********

Unable to retrieve js/css dependencies for article 'Sadist (band)': internal_api_error_Wikimedia\RequestTimeout\RequestTimeoutException

**********
```

Upstream issue: https://github.com/openzim/mwoffliner/issues/2000

I disabled the recipe for now

**Comment by kelson42** [comment]
Duplicate of #1270

**Comment by benoit74** [comment]
It's the other way around, it #1270 the duplicate and it has been closed, so let's reopen this one

**Comment by benoit74** [comment]
Issue solved with move to ActionParse, but there is a new one linked to the move to libzim 9 : https://github.com/openzim/mwoffliner/issues/2333

**Comment by dm1sh** [comment]
Seems like the last scrapping started on July 25, 2025 is stuck. Last log was on 2025-08-01, more then 10 days ago and it was like "Not able to download content for ...". And in stderr at the same time: `[error] [2025-08-01T10:28:22.245Z] Error (InternalError: We encountered an internal error.  Please retry the operation again later.) while downloading the object ...`. Both URLs are somewhere at upload.wikimedia.org.
I think, troubleshooting or restart is necessary.

**Comment by benoit74** [comment]
Not speaking about the fact it was started with 1.16.0 which is known to have issues with `maxi` of WPEN ... just restarted with `dev` version

**Comment by kelson42** [comment]
Finally successful! https://browse.library.kiwix.org/viewer#wikipedia_en_all_maxi_2025-08/

**Comment by Markus-Rost** [comment]
> Finally successful! https://browse.library.kiwix.org/viewer#wikipedia_en_all_maxi_2025-08/

@kelson42 @benoit74 Looks like the link used in the library is broken and doesn't redirect to the custom main page for some reason:
https://browse.library.kiwix.org/viewer#wikipedia_en_all_maxi_2025-08

Other 2025-08 mwoffliner ZIMs seem to be fine, even when using custom main pages. Adding a `/` like in your link above does result in the correct redirect.

**Comment by kelson42** [comment]
@rgaudin Something on the library creation side?

**Comment by benoit74** [comment]
Nope, it's "only" a cache issue, see https://github.com/kiwix/operations/issues/459 ; works well with the trailing `/` now that I've purged the cache for this ZIM, waiting to fix properly the "no-slash", not sure why purge did not worked for this (or more exactly, why we do not purge this as well)

Nothing wrong on ZIM generation side, library generation side, we have issue in proper repo, so let's close this issue again, nothing left here.
